File: Java/program/Mahasiswa.java
// deklarasi kelas mahasiswa
// kelas mahasiswa adalah child dari kelas Sivitas
public class Mahasiswa extends Sivitas {
    // atribut private
    private String nim, prodi, fakultas;

    /* konstruktor */

    // konstruktor tanpa parameter
    public Mahasiswa() {
        // set isi atribut secara default
        this.nim = "";
        this.prodi = "";
        this.fakultas = "";
    }

    /* Getter dan Setter */

    // Get nim
    public String getNim() {
        return this.nim;
    }

    // set nim
    public void setNim(String nim) {
        this.nim = nim;
    }

    // get prodi
    public String getProdi() {
        return this.prodi;
    }

    // set prodi
    public void setProdi(String prodi) {
        this.prodi = prodi;
    }

    // get fakultas
    public String getFakultas() {
        return this.fakultas;
    }

    // set fakultas
    public void setFakultas(String fakultas) {
        this.fakultas = fakultas;
    }
}

File: Python/program/Mahasiswa.py
# import class file
from Sivitas import Sivitas

# class declaration


class Mahasiswa (Sivitas):

    # atribut private
    __nim = ""
    __prodi = ""
    __fakultas = ""

   # konstruktor
    def __init__(self, nim="", prodi="", fakultas=""):
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    # Getter and Setter

    def get_nim(self):
        return self.__nim

    def set_nim(self, nim):
        self.__nim = nim

    def get_prodi(self):
        return self.__prodi

    def set_prodi(self, prodi):
        self.__prodi = prodi

    def get_fakultas(self):
        return self.__fakultas

    def set_fakultas(self, fakultas):
        self.__fakultas = fakultas
